core/logger/ocr_wrapper.go

```diff
@@ -12,6 +12,7 @@ type ocrWrapper struct {
 	saveError func(string)
 }
 
+// Deprecated: use github.com/smartcontractkit/chainlink-relay/pkg/logger.NewOCRWrapper instead.
```

Comment:
why not remove it now?
maybe add a ticket reference here and details in a ticket about what blocks deleting it

Reply:
Because it is exported API. In reality, we should never be deleting any exported API (or at least leave a grace period where old and new are both compatible), but `package logger` especially important since it is imported by many other things.